pkg/instrumentation/helper.go

```diff
@@ -25,3 +30,21 @@ func IsInitContainerMissing(pod corev1.Pod) bool {
 	}
 	return true
 }
+
+// Check if opentelemetry-auto-instrumentation volume is on the list.
+func IsOtAIVolumeMissing(volumeMounts []corev1.VolumeMount, logger logr.Logger) bool {
```

Comment:
Could you please include tests that will validate these new functions?

Reply:
Yeah, sure :) I forgot about it. Thanks!